metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py

```diff
@@ -529,7 +568,9 @@ def _map_azure_ad_group(self, azure_ad_group):
             self.config.azure_ad_response_to_groupname_regex,
         )
         if not self.config.groups_pattern.allowed(group_name):
-            self.report.report_filtered(f"{corp_group_urn}")
+            # lintFix giving error: "StatefulIngestionReport" has no attribute "report_filtered"
```

Comment:
Add a type annotation for the report type in this class. At the top of the class, add `report: AzureSourceReport` or similar

Do not use type ignore here

Reply:
done